frontend-react/src/components/Table/TableFilters.tsx

```diff
@@ -18,23 +18,19 @@ export enum StyleClass {
     DATE_CONTAINER = "date-picker-container tablet:grid-col",
 }
 
-export enum FilterName {
-    START_RANGE = "start-range",
-    END_RANGE = "end-range",
-    CURSOR = "cursor",
-    SORT_ORDER = "sort-order",
-    PAGE_SIZE = "page-size",
-}
-
 interface SubmissionFilterProps {
     filterManager: FilterManager;
     cursorManager?: CursorManager;
+    onFilterClick?: any;
```

Comment:
Let's try not to add any `any`s in our types and type this function:

```ts
// untested!
onFilterClick?: ({ from: string, to: string }) => void
```

Reply:
We could also add a custom type for the parameter structure here!  Might be overkill, but up to you!